src/main/java/io/jsonwebtoken/impl/DefaultJwtParser.java

```diff
@@ -59,7 +59,7 @@
     private static final String ISO_8601_FORMAT = "yyyy-MM-dd'T'HH:mm:ss'Z'";
     private static final int MILLISECONDS_PER_SECOND = 1000;
 
-    private ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
I agree with @lhazlewood 
It'd be better to be able to set the ObjectMapper. That being said, it is currently not the case and I would say it would be better to be consistant throught the different classes.

If the static is added though, please rename to OBJECT_MAPPER